ghost/admin/app/styles/layouts/members.css

```diff
@@ -2564,9 +2564,10 @@ p.gh-members-import-errordetail:first-of-type {
 .gh-cp-membertier-attribution .gh-tier-card-header {
     display: flex;
     align-items: flex-start;
+    position: relative;
 }
 
-.gh-cp-membertier-attribution .gh-tier-card-header:nth-of-type(2) {
+.gh-cp-membertier-attribution .gh-tier-card-header:not(:first-child) {
```

Comment:
This change adds support for more than 2 subscriptions in a list in admin (line and margin was missing)